src/gc/unix/gcenv.unix.cpp

```diff
@@ -417,18 +429,99 @@ size_t GCToOSInterface::GetLargestOnDieCacheSize(bool trueSize)
 //  A process affinity mask is a subset of the system affinity mask. A process is only allowed
 //  to run on the processors configured into a system. Therefore, the process affinity mask cannot
 //  specify a 1 bit for a processor when the system affinity mask specifies a 0 bit for that processor.
-bool GCToOSInterface::GetCurrentProcessAffinityMask(uintptr_t* processMask, uintptr_t* systemMask)
+bool GCToOSInterface::GetCurrentProcessAffinityMask(uintptr_t* lpProcessAffinityMask, uintptr_t* lpSystemAffinityMask)
```

Comment:
I don't think corert (FEATURE_REDHAWK?) does numa so I added non-NUMA GetCurrentProcessAffinityMask and GetCurrentProcessCpuCount implementations.